pgcli/completion_refresher.py

```diff
@@ -84,6 +84,10 @@ def _bg_refresh(self, pgexecute, special, callbacks, history=None, settings=None
         for callback in callbacks:
             callback(completer)
 
+        if not settings.get("single_connection"):
+            # close connection established with pgexecute.copy()
+            executor.conn.close()
```

Comment:
Do you think it makes sense to add a check for `if executor.conn:` here? Or perhaps a try/except?

Reply:
agreed, `if executor.conn` won't hurt

Reply:
@gantsevdenis so go ahead and add it. :) I'm waiting for that to merge the PR.

Reply:
yup, I did it here https://github.com/dbcli/pgcli/pull/1152/commits/b1b62c96a58a0baca29f85b89f38224f4bbb6a54 :)
`if not settings.get("single_connection") and executor.conn:`